apps/discord/discord.talon

```diff
@@ -0,0 +1,20 @@
+app: discord
+-
+# Navigation: Channels
+[channel] mentions last: action(user.discord_mentions_last)
+[channel] mentions next: action(user.discord_mentions_next)
+oldest unread: action(user.discord_oldest_unread)
+
+# UI
+toggle pins: action(user.discord_toggle_pins)
```

Comment:
I'm also not sure if we decided that we would abstract OS differences this way, because I know aegis doesn't like that it becomes harder to tell what commands actually do. But I guess if we are going to accept this, we should consider making an issue to convert the others... but I want to say we originally decided against this and favored duplication cross OS-specific set of actions? @knausj85 do you remember what the preference is, if we made one?

Reply:
Doesn't this need to be 

```
oldest unread:user.discord_oldest_unread()
```

to work? The syntax above is for implementing actions, or did I just miss a memo?

Other than that, there are two ways to support multiple OSes, @fidgetingbits 
- As emily's done here
- implementing the actions in .py

We haven't established any preference, and the repo is a mixed bag. The tradeoff with implementing things in .py: it's kinda out of reach for non-programmers. I find myself firmly on the fence, so I'm fine with either approach generally. Perhaps something to discuss elsewhere.

Reply:
> Doesn't this need to be

Yikes, sure does! Last-moment edit I failed to test thoroughly. Thanks, I'll fix it right up.

Reply:
Pushed a fix to the action calls.